packages/core/src/App/Containers/Onboarding/onboarding.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const Onboarding = () => <></>;
```

Comment:
does this component used anywhere?

Reply:
It was added as Container for new layout, since we are not implementing it I removed it for now